habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/util/PoetryUtil.java

```diff
@@ -0,0 +1,76 @@
+package org.technologybrewery.habushu.util;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Common utility methods for handling TOML Poetry.
+ */
+public final class PoetryUtil {
+    /**
+     * Specifies the semver compliant requirement for the version of Poetry that
+     * must be installed and available for Habushu to use.
+     */
+    public static final String POETRY_VERSION_REQUIREMENT = "^1.5.0";
+
+    /**
+     * Specifies the semver compliant requirement for the version of Poetry-core that
+     * must be installed and available for Habushu to use.
+     */
+    public static final String POETRY_CORE_VERSION_REQUIREMENT = "^1.6.0";
+
+    /**
```

Comment:
S: these don't seem like Poetry related Util methods, maybe they belong elsewhere?

Reply:
I am thinking of adding them to the TomlUtil class as these helper methods can be used to extract major and minor version.